packages/gp2-components/src/templates/__tests__/OnboardWelcome.test.tsx

```diff
@@ -0,0 +1,16 @@
+import { render, screen } from '@testing-library/react';
+import OnboardWelcome from '../OnboardWelcome';
+
+it('renders the header', () => {
+  render(<OnboardWelcome />);
+  expect(screen.getAllByRole('heading')[0]).toBeVisible();
```

Comment:
This is kind of lame and brittle if someone adds another title or something. Use get by text and then check that its a H1 or whatever

Reply:
could you do:

```
expect(screen.getByRole('heading', {name: /the-heading/i)).toBeVisible();
```